containers/ddev-php-base/Dockerfile

```diff
@@ -98,9 +96,14 @@ RUN apt-get -qq install --no-install-recommends --no-install-suggests -y \
     jq \
     msmtp \
     nginx \
-    nodejs \
     sqlite3
 
+RUN curl -fsSL https://raw.githubusercontent.com/tj/n/master/bin/n | bash -s "${NODE_VERSION}"
```

Comment:
You could just save the curl download rather than piping it into bash, if that is a good enough install for your purposes. Basic idea:

```
curl -fsSL https://raw.githubusercontent.com/tj/n/master/bin/n -o /usr/local/bin/n
chmod a+x /usr/local/bin/n
n install "${NODE_VERSION}"
```

Longer explanation: https://github.com/tj/n/issues/809#issuecomment-2264129789

Reply:
If @shadowspawn says it, it must be true! (thanks for being here!) @stasadev had gone the direction of the documented `n` install-with-bash technique, and it required `make` to be used.

Reply:
Thank you, @shadowspawn.

I will change this instruction to `n` when we prepare the next release of DDEV.